spec/features/user_sessions_spec.rb

```diff
@@ -43,27 +32,25 @@
     expect(page).to have_content('I told you kids to get out of here!')
   end
 
-  # it 'should redirect the user to their finish page after sign up' do
-  #   visit new_user_registration_path
-  #   fill_in :user_display_name, with: 'Rick'
-  #   fill_in :user_password, with: 'password123'
-  #   fill_in :user_email, with: 'm@il.com'
-  #   check :user_agree
-  #   click_button 'Create account'
-  #   expect(page).to have_content("Welcome Rick")
-  # end
-
-  # it 'should redirect the user to the page they were viewing after sign up' do
-  #   visit "/guides/new"
-  #   see ("You need to sign in or sign up before continuing.")
-  #   click_link "Sign up"
-  #   fill_in :user_display_name, with: 'Rick'
-  #   fill_in :user_password, with: 'password123'
-  #   fill_in :user_email, with: 'm@il.com'
-  #   check :user_agree
-  #   click_button 'Create account'
-  #   expect(page).to have_content(I18n::t('guides.new.new_guide_steps.create_a_growing_guide'))
-  # end
+  it 'should redirect the user to their finish page after sign up' do
+    visit new_user_registration_path
+    fill_in :user_display_name, with: 'Rick'
+    fill_in :user_password, with: 'password123'
+    fill_in :user_email, with: 'm@il.com'
+    click_button 'Join OpenFarm'
+    expect(page).to have_content('Thanks for joining!')
+  end
+
+  it 'should redirect the user to the page they were viewing after sign up' do
+    visit "/guides/new"
+    see ("You need to sign in or sign up before continuing.")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.